x-pack/platform/plugins/shared/entity_manager/server/lib/v2/queries/entity_instances.ts

```diff
@@ -46,7 +46,7 @@ const dslFilter = ({
 const statsCommand = ({ source }: { source: EntitySourceDefinition }) => {
   const aggs = source.metadata_fields
     .filter((field) => !source.identity_fields.some((idField) => idField === field))
-    .map((field) => `${field} = VALUES(${asKeyword(field)})`);
+    .map((field) => `${field} = TOP(${asKeyword(field)}, 10, "ASC")`);
```

Comment:
this was reverted by a bad merge in https://github.com/elastic/kibana/pull/203605